### DISCUSSION TITLE
Welcome to RLHD Discussions!

### DISCUSSION THREAD

**Discussion by Mark7625** [117HD/RLHD#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hugocowan** [comment]
Hey, I'm you're average Runelite enjoyer, and I'm also a javascript web developer too. I'm just curious, which is the main repo for 117 HD? It seems to be RS117/RLHD, but this fork is getting a lot of pull requests and merges. Is work for this fork going to be fed back into the main repo, or is this the main one now?

**Reply by aHooder** [reply]
We forked the RLHD repository on July 1st, and 117HD/RLHD is the new fork which we'll be using going forward. The remaining open pull requests on the old repository will probably get moved over soon ish. I'm hoping to submit an update to the plugin hub with some miscellaneous fixes and a port to LWJGL in the next coming days, though it may take longer to get reviewed by a RuneLite team member.

**Reply by hugocowan** [reply]
Yeah that port sounds very interesting. Would that help with rendering performance as an extra Q for you?

Good to know, thanks for clarifying. I'll watch progress on this with great interest, I'm impressed by the number of contributors.

**Reply by aHooder** [reply]
Only minimally, if at all on most systems. It does sound like M1 Macs will benefit quite a lot from it though.

**Reply by xxEzri** [reply]
> Yeah that port sounds very interesting. Would that help with rendering performance as an extra Q for you?
> 
> Good to know, thanks for clarifying. I'll watch progress on this with great interest, I'm impressed by the number of contributors.

Most of the chat is on the 117hd discord server btw

**Comment by j-meds** [comment]
Things to add to task or project @aHooder 5-7 are just extras.

1. Need to convert ObjectProps to JSON
2. Add JSON tooltip/typeahead in the ObjectProp JSON using enums(After implemetation)) ![image](https://user-images.githubusercontent.com/12689117/183512946-7eea5aff-f8f6-4fa0-b1c1-a7dda546b1f5.png)
3. Add JSON tooltip/typeahead in the Light JSON using Enums
4. Add Dev Panel to support appending to ObjectProperties Json.. Optional Add Light.json text box too. For now a text area works, future it could support files
5. Tile Info moved to dev panel @aHooder Approve of this? Checkbox
6. Dev Panel Extras: Add Object texture info 
-  list the id and applied textures, would allow for easy overrides in the json override textarea
- shortcut to copy JSON entry that modified texture, copy/paste in textarea to adjust in real time
7. Dev Panel Extras: Add Light info overlay info, list the light info,
